config/locales/zh-CN.yml

```diff
@@ -377,28 +387,29 @@ zh-CN:
     admin:
       tab:
         configuration: "配置"
-        option_types:
+        option_types: "选项类型"
         orders: "订单"
         overview: "概览"
         products: "产品"
         promotions: "促销"
-        properties:
-        prototypes:
+        promotion_categories: "促销分类"
```

Comment:
Think you correct here actually, the tab key seems to be absent all over except from `:ru` and `:pt_BR`